src/components/SidePanels/OutlinerPanel/OutlinerPanel.tsx

```diff
@@ -81,10 +86,19 @@ const OutlinerStandAlone: FC<OutlinerPanelProps> = ({ regions }) => {
         onFilterChange={onFilterChange}
       />
       {regions?.regions?.length > 0 ? (
-        <OutlinerTree
-          regions={regions}
-          selectedKeys={regions.selection.keys}
-        />
+        <>
+          <OutlinerTree
+            regions={regions}
+            selectedKeys={regions.selection.keys}
+          />
+          {hiddenRegions() > 0 && (
```

Comment:
please do the the calculation in a spot above the return then reuse the const. 
`const hiddenRegionCount = hiddenRegions();`
this way the callback gets fired one time then reuse that value as needed

Reply:
im over thinking this - useMemo whould work better here